src/host/screenInfo.cpp

```diff
@@ -680,7 +693,10 @@ VOID SCREEN_INFORMATION::InternalUpdateScrollBars()
     }
 
     // Fire off an event to let accessibility apps know the layout has changed.
-    _pAccessibilityNotifier->NotifyConsoleLayoutEvent();
+    if (_pAccessibilityNotifier)

```

Comment:
sometimes you use `HasAccessibilityEventing()` and sometimes you use `_pAcc...` _even inside `SCREEN_INFORMATION`_

Reply:
I guess one is directly guarding member access, and the other is guarding calling a more complex function. It's okay, but it did make me double-take.